appendices/rdp-controller.md

```diff
@@ -0,0 +1,35 @@
+# RDP Controller
+
+RDP stands for Remote Desktop Protocol and is the foundation for remote desktop connections in Windows. With RDP you can connect to a remote computer running Windows and use it as if it was your local machine.
+
+Our RDP Controller is a tool for running G1ANT.Robot on a remote PC: you just connect to a remote computer with the standard Remote Desktop Connection app in Windows, start a local robot script on this connected system, and then you can minimize or even close the remote connection. The script will continue the execution uninterrupted.
+
+In order to start using the RDP Controller, you have to follow these steps:
+
+1. Download and run [this file]() on your local Windows system, answering *Yes* to the security warning message.
+
+2. Launch Remote Desktop Connection tool in Windows: type `remote` in the Windows Search box (*Windows 7:* click Start Menu, then use the search box; *Windows 8 and above:* click the Search icon on the taskbar) and click the name of the app in the search results. You can also select the tool in the Accessories (Windows 7) or Windows Accessories (Windows 8 and above) folder of the Start menu (click All Programs first in case of Windows 7).
+
+   ![](G1ANT.Manual/-assets/rdc-start.jpg)
```

Comment:
incorrect path, should be: `![](-assets/rdc-start.jpg)`

Reply:
```suggestion
   ![](../-assets/rdc-start.jpg)
```